### PR TITLE
LOC CHECKIN | dotnet/winforms master | 20190417

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by zsd4yr** [comment]
@v-zbsail `<target>`'s were removed. Whatever tooling you're using is removing these fields which are required. See comment https://github.com/dotnet/winforms/pull/661#pullrequestreview-221397557 from previous check-in.

This is the second check-in which has been dead on arrival due to these syntax errors.

File: src/System.Windows.Forms.Design/src/Resources/xlf/SR.cs.xlf [hunk 1]
```diff
@@ -49,54 +49,52 @@
...
       <trans-unit id="CommandSetDelete">
         <source>Delete {0} components</source>
-        <target state="new">Delete {0} components</target>
+        <target state="translated">Odstranění součástí (počet: {0})</target>
         <note />
       </trans-unit>
       <trans-unit id="CommandSetError">
         <source>An error occurred while processing this command.
       {0}</source>
-        <target state="new">An error occurred while processing this command.
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design/src/Resources/xlf/SR.cs.xlf]: example

File: src/System.Windows.Forms.Design/src/Resources/xlf/SR.cs.xlf [hunk 2]
```diff
@@ -206,62 +202,56 @@
       {1}
 
       Stack trace:{2}</source>
-        <target state="new">The control {0} has thrown an unhandled exception in the designer and has been disabled.
```
Comment (zsd4yr) [inline src/System.Windows.Forms.Design/src/Resources/xlf/SR.cs.xlf]: example

Comment (janbaltus) [reply src/System.Windows.Forms.Design/src/Resources/xlf/SR.cs.xlf]: @zsd4yr  we updated PR with postprocess and target tag is added back.

**Pull request merged**